.pre-commit-config.yaml

```diff
@@ -1,6 +1,6 @@
 repos:
   - repo: https://github.com/antonbabenko/pre-commit-terraform
-    rev: "v1.92.1"
+    rev: "v1.97.0"
```

Comment:
You should also update the terraform-docs version (see line 9). I think v0.19.0 has been out for a while).

Reply:
The terraform-docs version has been updated.